spec/support/fixtures/test_models.rb

```diff
@@ -4,6 +4,8 @@ def self.table_name_prefix
   end
 
   class ClientRequest < ActiveRecord::Base
+    belongs_to :approving_official, class_name: User
```

Comment:
this class `ClientRequest` is only for testing and I added the association so that we didn't have to test `NCR::WorkOrder` explicitly. I.e. this is an example of an association.